lib/Ohmbrewer_Relay.cpp

```diff
@@ -99,12 +152,25 @@ bool Ohmbrewer::Relay::isOff() const {
 
 /**
  * Performs the Equipment's current task. Expect to use this during loop().
+ * Switches the power pin on or off for a generic relay, subclasses can utilize the controlPin
  * This function is called by work().
  * @returns The time taken to run the method
  */
 int Ohmbrewer::Relay::doWork() {
-    // TODO: Implement Relay::doWork
-    return -1;
+    int startTime = millis();
```

Comment:
If this commit has your doWork work in it and you merge it into your current branch for getting doWork working, you might have a bad time... Just a warning. You might want to make this part of the work more atomic. Your call.